app/models.py

```diff
@@ -0,0 +1,71 @@
+
+from app import db
+from passlib.apps import custom_app_context as pwd_context
```

Comment:
You can group your imports properly by adhering to PEP8 style guide, check this link: https://www.python.org/dev/peps/pep-0008/#imports